udata_front/theme/gouvfr/assets/less/components/table.less

```diff
@@ -22,3 +22,14 @@
 .fr-table__shadow:before {
     --table-offset: 0;
 }
+
+.fr-table:has(thead th .fr-btn) {
+    tbody td {
+        .fr-py-2w;
+        .fr-px-5v;
+    }
+    th {
+        padding: 0.75rem 0.5rem;
```

Comment:
I think this get overridden?
I'm still having alignment issue as shown in issue I think.

Reply:
Not working in Firefox, I'll find another way

Reply:
This is done with a new class `fr-table--dense` now